### PR TITLE
community/gegl: upgrade to 0.4.12

### PR DESCRIPTION
**Pull request by J0WI**
Required for GIMP 2.10.8

### CODE REVIEW

File: community/gegl/APKBUILD [hunk 1]
```diff
@@ -24,8 +24,14 @@ build() {
 	make
 }
 
+package() {
```
Comment (andypost) [inline community/gegl/APKBUILD]: I bet you mean `check()` )

File: community/gegl/APKBUILD [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gegl
-pkgver=0.4.8
+pkgver=0.4.12
 pkgrel=0
 pkgdesc="Graph based image processing framework"
-url="http://www.gegl.org/"
+url="https://www.gegl.org/"
```
Comment (andypost) [inline community/gegl/APKBUILD]: this URL does not work

File: community/gegl/APKBUILD [hunk 3]
```diff
@@ -24,8 +24,14 @@ build() {
 	make
 }
 
+check() {
+	cd "$builddir"
+	make test
```
Comment (andypost) [inline community/gegl/APKBUILD]: Surely check should be different
```
make: *** No rule to make target 'test'.  Stop.
>>> ERROR: gegl: check failed
```

**Comment by J0WI** [comment]
Fixed.

**Comment by J0WI** [comment]
Fixed in d98160bfe9f9ac36fa3b7de181e36ee5fb1be660

**Pull request closed without merging**